src/index.js

```diff
@@ -129,8 +133,9 @@ function executePayment (_subpayments, params) {
       notary: params.notary,
       notaryPublicKey: params.notaryPublicKey
     }
-    const executionCondition = conditionUtils.getExecutionCondition(conditionParams)
-    const cancellationCondition = isAtomic && conditionUtils.getCancellationCondition(conditionParams)
+
+    const executionCondition = params.executionCondition || conditionUtils.getExecutionCondition(conditionParams)
+    const cancellationCondition = isAtomic && (params.cancellationCondition || conditionUtils.getCancellationCondition(conditionParams))
```

Comment:
`cancellationCondition` will be `false` when `isAtomic` is `false` which means it can take on two different non-null types. I think the ternary operator is probably preferable for situations like this so you can set it to undefined explicitly.